apps/api/tests/utils.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from rest_framework.test import APITestCase
+
+from api.jwt_auth.views import JWTKeyAuthentication
+from api.tests.test_jwt_auth import JWTAuthTester
+from users.models import UserProfile
+
+
+class APIAuthTestCase(APITestCase, JWTAuthTester):
+
+    def create_api_user(self):
+        self.user = UserProfile.objects.create(
+            email='a@m.o',
+            read_dev_agreement=datetime.today()
+        )
+        self.api_key = self.create_api_key(self.user, str(self.user.pk) + ':f')
+
+    def authorization(self):
+        """
+        Creates a suitable JWT auth token.
+        """
+        token = self.create_auth_token(self.api_key.user, self.api_key.key,
+                                       self.api_key.secret)
+        return 'JWT {}'.format(token)
+
+    def get(self, url):
+        return self.client.get(url, HTTP_AUTHORIZATION=self.authorization())
+
+    def auth_required(self, cls):
+        """
+        Tests that the JWT Auth class is on the class, without having
+        to do a full request response cycle.
+        """
+        assert cls.authentication_classes == [JWTKeyAuthentication]
+
+    def verbs_allowed(self, cls, verbs):
+        """
+        Tests that the verbs you expect on the class are present and no more.
+        Options is added if you don't pass it.
+        """
+        verbs = set([v.upper() for v in verbs])
```

Comment:
The `[]` shouldn't be required. I'm not sure if there's a benefit one way or the other though.